adminlogin.aspx.cs

```diff
@@ -9,9 +11,45 @@ namespace LogBook
 {
     public partial class adminlogin : System.Web.UI.Page
     {
+        string strcon = ConfigurationManager.ConnectionStrings["con"].ConnectionString;
+
         protected void Page_Load(object sender, EventArgs e)
         {
 
         }
+
+        protected void Button1_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                SqlConnection con = new SqlConnection(strcon);
+                if (con.State == System.Data.ConnectionState.Closed)
+                {
+                    con.Open();
+                }
+                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
+                SqlDataReader dr = cmd.ExecuteReader();
+                if (dr.HasRows)
+                {
+                    while (dr.Read())
+                    {
+                        //Response.Write("<script>alert('" + dr.GetValue(0).ToString() + "');</script>");
+                        Session["username"] = dr.GetValue(0).ToString();
+                        Session["fullname"] = dr.GetValue(2).ToString();
+                        Session["role"] = "admin";
+                        //Session["status"] = dr.GetValue(9).ToString();
+                    }
+                    Response.Redirect("homepage.aspx");
+                }
+                else
+                {
+                    Response.Write("<script>alert('Invalid credentials');</script>");
+                }
+            }
+            catch (Exception ex)
+            {
+                Response.Write("<script>alert('"+ex.Message+"');</script>");
```

Comment:
### Don't use '+' for combining strings

Same comment as before. I would rather you use $@"{ex,Message}"; than '+' to combine strings. 
```suggestion
string errMsg = ex.Message;
                Response.Write($@"<script>alert({errMsg});</script>");
```